packages/good-design/src/advanced/socialshare/SocialShareWidget.ts

```diff
@@ -0,0 +1,444 @@
+// Import shared configuration
```

Comment:
This widget does not make sense, its not a compatible component within react-native.